include/khmer/_cpy_hashtable.hh

```diff
@@ -49,20 +49,38 @@ hashtable_consume_seqfile(khmer_KHashtable_Object * me, PyObject * args);
 
 
 PyObject *
+hashtable_consume_seqfile_with_reads_parser(khmer_KHashtable_Object * me,
+        PyObject * args);
+
+
+PyObject *
 hashtable_consume_seqfile_banding(khmer_KHashtable_Object * me, PyObject * args);
 
 
 PyObject *
+hashtable_consume_seqfile_banding_with_reads_parser(
+    khmer_KHashtable_Object * me, PyObject * args
+);
+
+
+PyObject *
 hashtable_consume_seqfile_with_mask(khmer_KHashtable_Object * me, PyObject * args);
 
 
 PyObject *
+hashtable_consume_seqfile_with_mask_with_reads_parser(
+    khmer_KHashtable_Object * me, PyObject * args
+);
+
+
+PyObject *
 hashtable_consume_seqfile_banding_with_mask(khmer_KHashtable_Object * me, PyObject * args);
 
 
 PyObject *
-hashtable_consume_seqfile_with_reads_parser(khmer_KHashtable_Object * me,
-        PyObject * args);
```

Comment:
Moved this function up for consistency's sake: for each overloaded method, the version that accepts a string is first, and the version that accepts a parser is second.